nspawn

```diff
@@ -29,10 +29,26 @@ set -e
 VERSION="0.2"
 STATIC_NAME="Nspawn (https://nspawn.org)"
 
+#
+# Defaults
+#
+
 BASEURL="https://hub.nspawn.org/storage"
 LISTURL="https://hub.nspawn.org/storage/list.txt"
 KEYLOCATION="https://hub.nspawn.org/storage/masterkey.pgp"
 
+#
+# Get vars from local configs if they exist - overriding the defaults
+# Last found config file wins
+# e.g. can be used for local mirrors without changing this file
+#
+
+LOCAL_CONFIGS=( "/etc/nspawn.conf" "$( dirname "${BASH_SOURCE[0]}" )/nspawn.conf")
+
+for local_config in ${LOCAL_CONFIGS[@]}; do
```

Comment:
Double quote array expansions to avoid re-splitting elements.
```suggestion
for local_config in "${LOCAL_CONFIGS[@]}"; do
```

Reply:
fixed. thanks for the hint